testsuite/integration/basic/src/test/java/org/wildfly/test/integration/web/postcontstruct/beanjar/SimpleBean.java

```diff
@@ -0,0 +1,4 @@
+package org.wildfly.test.integration.web.postcontstruct.beanjar;
+
+public class SimpleBean {
```

Comment:
Nitpickers note: I'd add a bean defining annotation to this class (`@Dependent`/`@ApplicationScoped`/...).

Reply:
I probably would, too, but this is what the reporter provided in the reproducer, so I've tried to change that as little as possible.

Reply:
It makes no difference for the purpose of this test - it's just that you now rely on having `beans.xml` with explicitly stated `all` discovery mode whereas with added annotation, the test would work even with empty `beans.xml` or completely without it :)